source/features/clean-issue-filters.tsx

```diff
@@ -40,18 +30,30 @@ function removeParent(element?: Element): void {
 	}
 }
 
-async function init(): Promise<void> {
-	const {projects, milestones} = await getCounts();
+async function hideMilestones(): Promise<void> {
+	const hasNoMilestones = select('[data-selected-links^="repo_milestones"] .Counter')!.textContent!.trim() === '0';
```

Comment:
No negative variables. I think:

```suggestion
	const hasMilestones = parseInt(select('[data-selected-links^="repo_milestones"] .Counter')?.textContent, 10) > 0;
```

Reply:
Uch I had it like that..